django/contrib/admin/templates/admin/includes/fieldset.html

```diff
@@ -20,7 +20,13 @@
                             {% else %}
                                 {{ field.label_tag }}
                                 {% if field.is_readonly %}
-                                    <div class="readonly">{{ field.contents }}</div>
+                                    <div class="readonly">
+                                      {% if field.required_perm and field.required_perm not in perms %}
+                                        {{ field.contents|striptags }}
+                                      {% else %}
+                                        {{ field.contents }}
+                                      {% endif %}
+                                    </div>
```

Comment:
```suggestion
                                    <div class="readonly">{% if field.required_perm and field.required_perm not in perms %}{{ field.contents|striptags }}{% else %}{{ field.contents }}{% endif %}</div>
```
The tests appear to be new line sensitive